.dir-locals.el

```diff
@@ -0,0 +1 @@
+((haskell-mode . ((before-save-hook . (ormolu-format-buffer)))))
```

Comment:
This gives me a warning every time I visit a file in emacs.

Reply:
(I already have my own save hook setup for Ormolu)

Reply:
Right. How did you setup yours? It should be something local to the Carp directory (we don't want to format every codebase under the sun with ormolu).

Reply:
True, I did set it up globally (which is not great).
Maybe there's a way to avoid the warning?

Reply:
You can add it to the list of safe variables, but that's unsafe, somebody could add malicious code in a `.dir-locals.el` as a `before-save-hook`. There must be something better.

Reply:
Can you try now? 

Using `0` in .dir-locals should allow disabling it for a directory.